### DISCUSSION TITLE
Unrecognized index name wikipedia-dpr-100w.dpr-multi

### DISCUSSION THREAD

**Discussion by abdoelsayed2016** [castorini/pyserini#1694] [Q&A]
I tired to run experiment dpr 

`python -m pyserini.search.faiss --index wikipedia-dpr-100w.dpr-multi  --topics dpr-nq-test  --encoded-queries dpr_multi-nq-test --output runs/run.dpr.nq-test.multi.bf.trec --batch-size 36 --threads 12`

got error 

C:\Users\Abdo\.cache\pyserini\queries\query-embedding-dpr_multi-nq-test-20210419-9323ec.1791f1ed078beb3a00847f75023eb020 already exists, skipping download.
Initializing dpr_multi-nq-test...
Attempting to initialize pre-built index wikipedia-dpr-100w.dpr-multi.
Unrecognized index name wikipedia-dpr-100w.dpr-multi

anyone has solution where can i download this index ?

**Comment by lintool** [comment]
Which version of the code are you using? We've renamed this index recently, so you might be using an out-of-date version.

If you do a dev install of the latest master, these are the most up-to-date index names/versions: https://github.com/castorini/pyserini/blob/master/pyserini/prebuilt_index_info.py

**Comment by abdoelsayed2016** [comment]
using pyserini-0.19.0
and same example in Docs for DPR

**Comment by lintool** [comment]
Yes, please update to a later version.
